### ISSUE TITLE
Battery Powered RPI5+Hailo+Camera YOLO Solution

### ISSUE THREAD

**Issue by sourcesync** [hailo-ai/hailo-rpi5-examples#71]
Do you (or anyone in this forum) have any real world experiment running RPI5 + Hailo + Camera + YOLO battery powered?

If so, some questions I have for you:
* what frame rate did you get?
* how long one battery charge last running at a reasonable real-time frame rate?
* what parts do you recommend purchasing?  Amazon links would be useful.

I'm not looking for hypotheticals but actually real experience with a running system.

Thanks!

**Comment by Starman7312** [comment]
Yes I have, it runs ok. I think the frame rate seems the same, but obviously power runs out a bit quicker than normal (definitely 4 hours at least, if not more using a 20W, 10,000 mAh powerbank)

**Comment by Starman7312** [comment]
This is the power bank I use: https://thepihut.com/products/ansmann-10-000mah-type-c-18w-pd-power-bank

**Comment by ARDik596** [comment]
Right now I'm using hoverboard 36V Li-ion battarey with XL4016 DC-DC converter. I set it on 5.1V output and attached usb-c socket to it's output so now i can power whole RPI by any USB-C PD cable.
